### PR TITLE
Periodically run oc adm top nodes during testing

### PR DESCRIPTION
**Pull request by hamzy**
This will add an infinite loop which periodically runs the command:
    oc adm top nodes
and logs the output to a file in the artifacts directory.

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml [hunk 1]
```diff
@@ -302,8 +302,32 @@ objects:
...
+          # Close STDERR FD
+          exec 2<&-
+          # Open STDOUT as $LOG_FILE file for read and write.
+          exec 1<>${LOG_FILE}
+          # Redirect STDERR to STDOUT
+          exec 2>&1
+
+          while true
+          do
+            echo "***************************************************************"
```
Comment (jaypoulz) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:320]: Would it be possible to provide a call to "date" here as well. It's good to see how the resource allocation changes over the course of a run, but I think being able to match it up with key moments (or failures) in the run could be helpful as well.

**Review by jaypoulz — CHANGES_REQUESTED** [review]
Overall, I like the concept. Being able to see how stuff changes over the course of a run is useful for debugging. Once we have something to help us note the time, I think this is good to go.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml (hunk 1 above)
Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:320]: That's a good idea!  Will do...

**Review by jaypoulz — APPROVED** [review]
LGTM

**Pull request merged**